docs/install.rst

```diff
@@ -67,16 +86,23 @@ You may also need to install the `Gulp <http://gulpjs.com/>`_ cli globally::
 .. _run-python-tests:
 
 Run the tests
--------------
-
-.. Important::
-
-    We're working on fixing this, but for now you need the localization files for the tests to pass.
-    See the `Localization`_ section below for instructions on checking those out.
+=============
 
 Now that we have everything installed, let's make sure all of our tests pass.
 This will be important during development so that you can easily know when
-you've broken something with a change. You should still have your virtualenv
+you've broken something with a change.
+
+Docker
+------
+
+We manage our local docker environment with docker-compose. All you need to do here is run::
+
+    $ docker-compose run test
```

Comment:
Note for other reviewers: I initially got a bunch of errors when running the tests using this command, and needed to clean out my `.pyc` files using `find . -name \*.pyc -delete`. After doing this the tests passed OK.

Reply:
This is similar to the database issue in that I don't think may people would have .pyc files who are using docker, but it is possible. I can maybe put a .pyc deletion step in a script that this runs.